### PR TITLE
chore: Sync with Crowdin

### PR DESCRIPTION
**Pull request by goodhoko**

### CODE REVIEW

File: packages/suite-data/files/translations/en.json [hunk 1]
```diff
@@ -722,7 +722,7 @@
 	"TR_GAS_PRICE": "Gas price",
 	"TR_GAS_USED": "Gas used",
 	"TR_GATHERING_INFO": "Gathering information, please wait...",
-	"TR_GENERAL": "General",
+	"TR_GENERAL": "Application",
```
Comment (goodhoko) [inline packages/suite-data/files/translations/en.json:726]: This is suspicious.

File: packages/suite-data/files/translations/en.json [hunk 2]
```diff
@@ -742,7 +742,7 @@
 	"TR_GUIDE_CATEGORY_DICTIONARY": "Dictionary",
 	"TR_GUIDE_CATEGORY_TREZOR": "Trezor",
 	"TR_GUIDE_CATEGORY_TREZOR_SUITE": "Trezor Suite",
-	"TR_GUIDE_FEEDACK_OR_SUGGESTION": "Feedback or Suggestion",
+	"TR_GUIDE_FEEDACK_OR_SUGGESTION": "Report Bug  & Feedback",
```
Comment (goodhoko) [inline packages/suite-data/files/translations/en.json]: There's a double-space.

File: packages/suite-data/files/translations/en.json (hunk 1 above)
Comment (goodhoko) [reply packages/suite-data/files/translations/en.json:726]: OTOH, this key is used on a single place in the settings menu. So this is most probably fine.,

File: packages/suite-data/files/translations/en.json (hunk 2 above)
Comment (goodhoko) [reply packages/suite-data/files/translations/en.json]: Fixed in crowdin. Synced again and forcepushed. Resolving.

**Review by tsusanka — APPROVED** [review]

**Pull request merged**